java-impl/src/main/java/org/jetbrains/research/refactorinsight/data/methods/InvertConditionJavaHandler.java

```diff
@@ -0,0 +1,15 @@
+package org.jetbrains.research.refactorinsight.data.methods;
+
+import org.jetbrains.research.refactorinsight.data.JavaRefactoringHandler;
+import org.jetbrains.research.refactorinsight.data.RefactoringInfo;
+import org.refactoringminer.api.Refactoring;
+
+public class InvertConditionJavaHandler extends JavaRefactoringHandler {
+    @Override
+    public RefactoringInfo specify(Refactoring refactoring, RefactoringInfo info) {
+//        InvertConditionRefactoring ref = (InvertConditionRefactoring) refactoring;
```

Comment:
Do we plan to support this kind of refactoring?

Reply:
Yes, I plan to support these kinds of refactorings in the future. These handlers return null because the implementation requires a new RefactongMiner update.